test/test_NNBuilder.py

```diff
@@ -0,0 +1,17 @@
+from unittest import TestCase
+import torch
+import numpy as np
+from torchagents.utilities import NNBuilder
+
+class TestNNBuilder(TestCase):
+
+    def tearDown(self):
+        pass
+
+    def setUp(self):
+        self.config = {'input_size': np.random.randint(10), 'output_size': np.random.randint(10), 'hidden_layers': [128, 128, 256], 'output_function': torch.nn.Softmax}
```

Comment:
I think the random seed should be fixed here.

Reply:
Agree!